plugins/mocker/main.go

```
@@ -0,0 +1,1075 @@
+		}
+		return p.faker.Lorem().Sentence(wordCount)
+	case "uuid":
+		return p.faker.UUID().V4()
+	case "hex_color":
+		return p.faker.Color().Hex()
+	case "integer":
+		min, max := 1, 100 // defaults
+		if len(parts) > 1 {
+			params := strings.Split(parts[1], ",")
+			if len(params) >= 2 {
+				fmt.Sscanf(params[0], "%d", &min)
```

golangci-lint (1.64.8)

Error return value of `fmt.Sscanf` is not checked

(errcheck)